### PR TITLE
adding library challenge to pod 1 repo

### PR DESCRIPTION
**Pull request by Jason-Doze**
This one was pretty tricky, we were kinda there but not quite.  Interesting to have to add print to calling the function.

### CODE REVIEW

File: Jason/library_challenge.py [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+    },
+    {
+        'title': 'THE ROAD TO UNFREEDOM',
+        'author': 'Timothy Snyder',
+        'description': 'Snyder explores Russian attempts to influence Western democracies and the influence of philosopher Ivan Ilyin on Russian President Vladimir Putin and the Russian Federation in general.'
+    }
+]
+# 2.0
+# Describe the structure of the data in books_with_details. What types of data are nested within others? How do you know?
+# A series of dictionaries within a list. We know by the hard bracket indicating a list and crly brackets indicating a dictionary
```
Comment (Tyler-Toro) [inline Jason/library_challenge.py:161]: Short concise answer, like a coder

File: Jason/library_challenge.py [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+
+# 2.3
+# Create a function 'search_by_author' that returns the titles of books by an author
+# Parameters - author (string)
+# Return - author's books (list of strings)
+# Hint - You will need a for loop, if statement, .append() for this solution!
+
+def search_by_author(str):
+    author_titles = []
+    for i in books_with_details:
```
Comment (Tyler-Toro) [inline Jason/library_challenge.py:196]: function looks good. If you had to change the iterator ' i ' to a different name, what would you choose?

**Review by Tyler-Toro — APPROVED** [review]
Looking good. Whats been most helpful for getting code organized correct? Modules? Are you reading up info outside of JTC?  Trying to gain insight on how to help others

**Comment by Jason-Doze** [comment]
Whats been most helpful for getting code organized correct? Modules? Are you reading up info outside of JTC?

I really don’t know what I’m doing, I’m just throwing things together and hoping something works. Up until this tip calculator project, I haven’t been looking up anything outside our course work. For the tip calculator I’ve been looking on GitHub.

> If you had to change the iterator ' i ' to a different name, what would you choose?

> I think we had for author in previously before the rest of the pod got it work with this code.

Encrypted email based in Switzerland.

**Pull request merged**